cn.feishu.Feishu.yaml

```diff
@@ -0,0 +1,68 @@
+app-id: cn.feishu.Feishu
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "23.08"
+command: feishu
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=x11
+  # - --socket=wayland
+  - --socket=pulseaudio
+  # For ibus
+  # - --socket=session-bus
+  # - --socket=system-bus
+  # For camera
+  - --device=all
+  # For ptrace
+  - --allow=devel
+  - --filesystem=home
```

Comment:
why is this needed?

Reply:
Feishu would call ptrace for some reason, I have disable it and feishu still run well.

Reply:
I meant why is home access needed?

Reply:
Feishu is a collaboration and management platform integrating instant communication, I think users can share files and docs with each other easier if feishu can access home dir.